Dockerfile

```diff
@@ -28,10 +28,8 @@ RUN ./perf_build.sh
 # ubuntu 20.04
 FROM ubuntu@sha256:cf31af331f38d1d7158470e095b132acd126a7180a54f263d386da88eb681d93 AS bcc-builder
 
-RUN apt-get update
```

Comment:
Note: also needs to be updated in `pyi.Dockerfile` which is CentOS based :/

You're now trying to share the `pyperf_env.sh` script - you can try doing so, or clone just a script `scripts/build_libunwind.sh`

Reply:
Separating `libunwind` build to its own script makes sense